MREUnityRuntime/MREUnityRuntimeLib/RPC/RPCChannelInterface.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using MixedRealityExtension.Messaging.Payloads;
+using System;
+using System.Collections.Generic;
+
+namespace MixedRealityExtension.RPC
+{
+	public sealed class RPCChannelInterface
+	{
+		private Dictionary<string, RPCInterface> channelHandlers = new Dictionary<string, RPCInterface>();
+		private RPCInterface globalHandler;
+
```

Comment:
Should prefix these privates with '_' for consistency with out private field naming conventions.